flaml/automl/ml.py

```diff
@@ -678,18 +687,18 @@ def compute_estimator(
 
 
 def train_estimator(
-    config_dic,
+    config_dic: dict,
     X_train,
     y_train,
-    task,
-    estimator_name,
-    n_jobs=1,
-    estimator_class=None,
+    task: str,
+    estimator_name: str,
+    n_jobs: int = 1,
+    estimator_class: Optional[EstimatorSubclass] = None,
     budget=None,
-    fit_kwargs={},
+    fit_kwargs: dict = {},

```

Comment:
Optional

Reply:
Hey, thanks for all your feedbacks!
This was slightly tricky to handle. Take a look at my proposed solution in the updated version